src/Behat/Config/Filter/Filter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config\Filter;
+
+use Behat\Config\ConfigConverterInterface;
+use PhpParser\BuilderFactory;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Name\FullyQualified;
+
+class Filter implements FilterInterface, ConfigConverterInterface
```

Comment:
I created a base class for all named filters which implements most of the functionality of these filters